include/deal.II/numerics/matrix_tools.h

```diff
@@ -220,6 +220,9 @@ namespace MatrixCreator
    * Assemble the mass matrix. If no coefficient is given (i.e., if the
    * pointer to a function object is zero as it is by default), the
    * coefficient is taken as being constant and equal to one.
+   * In case you want to specify @p constraints and use the default argument
+   * for the coefficient you have to cast the nullptr, i.e.
+   * <code>const Function<dim> *const(nullptr)</code>.
```

Comment:
does this syntax actually work? Or does it have to be `(const Function<spacedim> *const)nullptr` or `static_cast<...>(nullptr)`?

I don't think "you have to cast the nullptr" is easy to understand. Maybe "...you have to specify the (unused) coefficient argument as <code>...</code>".

same elsewhere, of course.